utils/meta/open_data_network/open_data.py

```diff
@@ -0,0 +1,177 @@
+import csv
+import json
+from os.path import exists
+
+import requests
+
+# Categories, names, and tags used to filter out irrelevant data
+FILTERED_CATEGORIES = [
+    "building and facilities", "information", "administration & finance", "general government", "retirement systems",
+    "economy", "transportation", "growing economic opportunities", "environment", "licensing", "covid",
+    "culture and recreation", "public works", "tax", "city management and ethics", "thriving neighborhoods",
+    "infrastructure", "planning", "human services", "city government", "health and social services", "automobiles",
+    "neighborhood census data", "business", "housing", "service requests", "ptsd-repository", "fire and medical",
+    "geographic locations and boundaries", "services and amenities", "buildings", "transparency", "water", "surveys",
```

Comment:
`geographic locations and boundaries` can be helpful for understanding jurisdiction, `transparency` seems too broad to include here.

Reply:
> `geographic locations and boundaries` can be helpful for understanding jurisdiction, `transparency` seems too broad to include here.

@josh-chamberlain The `transparency` category only has one dataset belonging to it, which is this: [NYS Attorney Registration](https://data.ny.gov/Transparency/NYS-Attorney-Registrations/eqw2-r5nb). On second thought this could be relevant for a court scraper, so I'll remove it as a filter. The `geographic locations and boundaries` also only includes one dataset [Parcels with overlay attributes](https://data.sfgov.org/Geographic-Locations-and-Boundaries/Parcels-with-overlay-attributes/9grn-xjpx), upon further inspection it looks like it does contain police jurisdiction data (or at least a link to another dataset that contains it) so I'll remove that filter as well